rough_draft_aim2.md

```diff
@@ -0,0 +1,17 @@
+
+Human Cell Atlas' partnership with the Immunological Genome Project (immgenH) will result in gene expression-based immunocyte phenotyping at unprecedented resolution, making it well-suited for the problem of studying rheumatic/autoimmune disorders and for addressing the question of the interplay between tissue-resident immune cells and those cells infiltrating from peripheral blood in these disease states.
+
+We will apply Pathway-level Information ExtractoR (**PLIER**)([Mao ref](http://dx.doi.org/10.1101/116061)), which has been demonstrated to be effective in extracting cell type-specific signals from bulk gene expression data, to bulk autoimmune/rheumatic disease data. PLIER requires bulk expression data and prior knowledge as input. In this case, 
+we will encorporate cell subset and activation state information obtained from single-cell data as prior information (e.g., sets of genes that distinguish cell-types). (Note that this analysis framework is agnostic to the particular methodology used to generate low-dimensional representations of cell-types/states.) 
```

Comment:
I see, so the purpose of using PLIER is to deconvolute cell-types from bulk, using defining markers of cell type found from single cell computational approaches. Can you add a bit more to why PLIER is the method of choice here?